libs/langchain/langchain/agents/agent_toolkits/openapi/spec.py

```diff
@@ -97,6 +97,8 @@ def reduce_endpoint_docs(docs: dict) -> dict:
             ]
         if "200" in docs["responses"]:
             out["responses"] = docs["responses"]["200"]
+        if docs.get("requestBody"):
```

Comment:
Looks reasonable -- @hinthornw do you know if this was dropped intentionally (context length?)